testgrid/config/config.proto

```diff
@@ -117,6 +117,9 @@ message DashboardTab {
 
   // The URL template to visit when searching for changelists.
   LinkTemplate code_search_url_template = 12;
+
+  // A description paragraph to be displayed.
+  string description = 13;
```

Comment:
Personally I would also like to attach the description to the test group. Any way we can plumb that through?